### PR TITLE
feat(home-manager): init fcitx5 module

### PR DESCRIPTION
**Pull request by pluiedev**
Self-explanatory. :)

### CODE REVIEW

File: modules/home-manager/fcitx5.nix
[the remark this answers is not in the document]
Comment (pluiedev) [reply modules/home-manager/fcitx5.nix:1]: Ahhh... Didn't know that's a thing. Thanks!

[the remark this answers is not in the document]
Comment (isabelroses) [reply modules/home-manager/fcitx5.nix:1]: > Ahhh... Didn't know that's a thing. Thanks!

Sorry should have been clearer assertions are under `config`

```nix
config.assertions = [
  (lib.ctp.assertXdgEnabled "fcitx5")
];
```

[the remark this answers is not in the document]
Comment (pluiedev) [reply modules/home-manager/fcitx5.nix:1]: Oh yeah I forgot, haven't written a module in ages haha

**Review by getchoo — APPROVED** [review]
thanks :)

**Pull request merged**